frontend/include/chpl/parsing/parsing-queries.h

```diff
@@ -213,6 +213,33 @@ const std::vector<UniqueString>& moduleSearchPath(Context* context);
 void setModuleSearchPath(Context* context,
                          std::vector<UniqueString> searchPath);
 
+/**
+  Return a list of paths to be prepended to the internal module path. This is
+  likely to be empty unless set using --prepend-internal-module-dir
```

Comment:
I would mention either in these or in the big `setupModule...` mega function that these will be set implicitly when that is called if they are not already set.